helmfiles/tests/README.md

```diff
@@ -0,0 +1,128 @@
+# Tests
+
+These tests Dgraph delployments with a variety helm config values.  
+
+## Create Secrets Using Docker Container
+
+```bash
+## Docker environment for the `dgraph cert` command
+docker run --rm --detach \
+  --publish "8080:8080" \
+  --publish "9080:9080" \
+  --publish "8000:8000" \
+  --volume $PWD/dgraph_tls:/dgraph_tls \
+  --name "dgraph-certs" \
+  "dgraph/standalone:v21.03.0"
+
+## Copy script to into container
+docker cp \
+  ../../charts/dgraph/scripts/make_tls_secrets.sh \
+  dgraph-certs:/make_tls_secrets.sh
+
+## Generate TLS certs per each test
+TLS_TESTS="alpha-tls zero-tls"
+for TEST in $TLS_TESTS; do
+  docker exec -it dgraph-certs /make_tls_secrets.sh --release "test" \
+    --client "dgraphuser" --namespace dgraph-test-$TEST --zero --tls_dir /dgraph_tls/dgraph-test-$TEST
+done
+
+## Verify certs/keys hostnames
+for TEST in $TLS_TESTS; do
+  echo "Dgraph Alpha: Release[test], Namespace [dgraph-test-$TEST]:"
+  echo "-------------------------------------------------------------"  
+  docker exec -it dgraph-certs dgraph cert ls --dir /dgraph_tls/dgraph-test-$TEST/alpha | awk -F: '/Hosts/{gsub(/\[ ]+/, "", $2); print $2}' | tr , '\n'
+  echo "Dgraph Zero: Release[test], Namespace [dgraph-test-$TEST]:"
+  echo "-------------------------------------------------------------"
+  docker exec -it dgraph-certs dgraph cert ls --dir /dgraph_tls/dgraph-test-$TEST/zero | awk -F: '/Hosts/{gsub(/\[ ]+/, "", $2); print $2}' | tr , '\n'
+  printf "\n"
+done
+```
+
+## Run Tests
+
+This will deploy a dgraph cluster using different types of dgraph configuration.  Use the `--environment $TEST` flag to select the desired test.  
```

Comment:
```suggestion
This will deploy a Dgraph cluster using different types of Dgraph configuration.  Use the `--environment $TEST` flag to select the desired test.  
```